.eslintrc.json

```diff
@@ -41,13 +50,20 @@
         "allowNamedFunctions": true
       }
     ],
+    "camelcase": "off",
     "new-cap": "error",
     "no-var": 1,
     "react/jsx-uses-react": "error",
-    "react/jsx-uses-vars": "error"
+    "react/jsx-uses-vars": "error",
+    "react/jsx-indent": "off",
+    "no-mixed-spaces-and-tabs": "off",
```

Comment:
Shouldn't that be on if we want to have just spaces?

Reply:
Oh yea, I was using it for testing when I was trying to override the default react styles. This PR is mostly for feedback, so every suggestion is taking into account.